### ISSUE TITLE
Support "A3:A" style references?

### ISSUE THREAD

**Issue by cmcaine** [queryverse/ExcelReaders.jl#70]
In Excel, this refers to all the cells in the A column from row 3 till the bottom.

It would have been convenient for my use case to have this select from row 3 till the last row with any non-missing cells.

**Comment by davidanthoff** [comment]
Yes, good idea! I won't have time to implement this anytime soon, but a PR that does would be great.
